python/activator/activator.py

```diff
@@ -34,6 +34,7 @@
 
 from lsst.daf.butler import Butler
 from lsst.obs.base import Instrument
+from .logger import GCloudStructuredLogFormatter
```

Comment:
I have a question about the activator file, are there any unit tests for this? Should we be adding tests to cover the code thats been added?

Reply:
No, `activator.py` is very hard to unit test because the `google` import can't be resolved in the standard stack. I think @parejkoj mentioned some problems with unit-testing the Flask bits as well.

I don't see a better solution than gradually factoring all the non-Google code into other files, and then testing those.

Reply:
I'd advocate doing that refactor, and adding some flask-based mocks to test the web-interfacing portion. I'm pretty sure that Google has released some code to allow one to make a mock service to test against, but I do recall feeling like it would take me too long to figure it out during the original sprint.